editormd.js

```diff
@@ -3957,7 +3957,7 @@
             smartypants : true
         };
         
-		markdownDoc = new String(markdownDoc);
+	markdownDoc = new String(markdownDoc).toString();

```

Comment:
Maybe you can add two spaces to align code.

Reply:
It is already aligned in context, please check this line in the source file.

Reply:
Sorry, you are right.